plugins/git/octo.nix

```diff
@@ -0,0 +1,169 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "octo";
+  originalName = "octo.nvim";
+  defaultPackage = pkgs.vimPlugins.octo-nvim;
+
+  maintainers = [ helpers.maintainers.svl ];
+
+  extraPackages = [ pkgs.gh ];
+
+  settingsOptions = {
+    use_local_fs = helpers.defaultNullOpts.mkBool false ''
+      Use local files on right side of reviews.
+    '';
+
+    enable_builtin = helpers.defaultNullOpts.mkBool false ''
+      Shows a list of builtin actions when no action is provided.
+    '';
+
+    default_remote = helpers.defaultNullOpts.mkListOf types.str ''[ "upstream" "origin" ]'' ''
```

Comment:
```suggestion
    default_remote = helpers.defaultNullOpts.mkListOf types.str ["upstream" "origin"] ''
```

Reply:
it doesn't seem to show properly. are there some new changes?

https://github.com/nix-community/nixvim/pull/1541#discussion_r1611570714